tests/jobs/scala/src/main/scala/ProvidedPackages.scala

```diff
@@ -0,0 +1,42 @@
+import org.apache.spark.sql.SparkSession
+import com.google.common.math.IntMath
+import com.google.common.base.Verify
+
+/**
+  * Application that outputs the sum of numbers from 1 to N.
+  * Specifically we want to test whether a package (e.g. guava) is available by specifying in the packages flag in submit-args.
+  * E.g. 
+  * dcos spark run --submit-args="\
+        --packages=com.google.guava:guava:23.0 \
+        --class ProvidedPackages \
+        <jar> number"
+  */ 
+object ProvidedPackages {
+    def main(args: Array[String]): Unit = {
+        val AppName = "ProvidedPackages App"
```

Comment:
Let's keep the name the same as class name. And, please make the val name start with lower-case because it's not a class-level constant (according to scalastyle).

Reply:
Corrected the variable name